Configurations/10-main.conf

```diff
@@ -1483,6 +1483,16 @@ my %targets = (
         lib_cppflags     => "-DL_ENDIAN",
         sys_id           => "UEFI",
     },
+    "UEFI-x86" => {
+        inherit_from     => [ "UEFI" ],
+        asm_arch         => 'x86',
+        perlasm_scheme   => "win32n",
+    },
+    "UEFI-x86_64" => {
+        inherit_from     => [ "UEFI" ],
+        asm_arch         => 'x86_64',
+        perlasm_scheme   => "nasm",
```

Comment:
Why not "auto", like VC-WIN64A?

Reply:
EDK2 prescribes nasm for assembly files and this lets me trigger the ($flavour =~ /nasm/) check in the perl generator files.

Reply:
Okie